products/msexchange.md

```diff
@@ -11,22 +11,22 @@ releaseDateColumn: true
 
 releases:
 -   releaseCycle: "2019"
-    releaseLabel: "2019 CU12 Mar23SU"
+    releaseLabel: "2019 CU13 Jun23SU"
```

Comment:
This comment is outside the scope of this PR.

Shouldn't we use the naming `2019 CU13 SU1` instead ? I saw that on https://support.microsoft.com/en-us/topic/description-of-the-security-update-for-microsoft-exchange-server-2019-june-13-2023-kb5026261-c4abbd97-5afe-4229-9ecf-a0b7c0002a5d.

Reply:
In the past we have used the terms from https://learn.microsoft.com/exchange/new-features/build-numbers-and-release-dates.

Reply:
Ok, thanks for the information.